src/extension/exponent/exponentPlatform.ts

```diff
@@ -68,7 +71,12 @@ export class ExponentPlatform extends GeneralMobilePlatform {
                     return Q.reject<string>(reason);
                 })
                 .then(exponentUrl => {
-                    vscode.commands.executeCommand("vscode.previewHtml", vscode.Uri.parse(exponentUrl), 1, "Expo QR code");
+                    let exponentPage = vscode.window.createWebviewPanel("Expo QR Code", "Expo QR Code", vscode.ViewColumn.Two, {
+                         enableScripts: true,
```

Comment:
Please clarify, why do we need to enable scripts for this page? According to this docs, we shouldn't allow this unless it's really needed:
https://code.visualstudio.com/docs/extensions/webview#_limit-capabilities

Reply:
Looks like webpage is working without this options, so I deleted them.